projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -179,17 +152,15 @@ export class ClrDatagridWidgetObject<T> extends BaseWidgetObject<T> {
      * @param col 0-based index of column
      */
     getColumn(col: number): T {
-        // return this.locatorDriver.get(`${Css.COLUMN}:nth-of-type(${col + 1})`).unwrap();
         return this.locatorForCssSelectors(`${Css.COLUMN}:nth-of-type(${col + 1})`)();
     }
 
     /**
-     * Returns input element in the given row
+     * Returns datagrid select element in the given row
```

Comment:
It's unclear what `datagrid select` is supposed to mean. It sounds like it's going to grab a `<select>` element. This has the exact same code as `getSingleSelectionRadioLabel`, maybe that function can be improved to be just `getSelectionElementForRow(number)` and it works for both single and multiple select.